src/dosbox.cpp

```diff
@@ -454,17 +456,17 @@ void DOSBOX_Init(void) {
 	                "320x200 or 640x400; where as square-pixel modes, such as 640x480\n"
 	                "and 800x600, will be displayed as-is.");
 
-	Pmulti = secprop->Add_multi("scaler", Property::Changeable::Always, " ");
-	Pmulti->SetValue("normal2x");
-	Pmulti->Set_help("Scaler used to enlarge/enhance low resolution modes.\n"
+	pmulti = secprop->Add_multi("scaler", always, " ");
+	pmulti->SetValue("none");
+	pmulti->Set_help("Scaler used to enlarge/enhance low resolution modes.\n"
```

Comment:
Very welcome default change!  With SDL2's arbitrary HW hardware scaling (with `openglnb` or `texturepp`), I think this eliminate the need for all future `normal*x` scalers.  I suspect this also makes the `sharp` GL shader redundant too (so `shader = none` could also be the default?).

Reply:
Perhaps we can make SDL2 arbitrary scaling working, but that would be another feature to be implemented later (and at another significant development and testing cost :(). I consider it out of scope of the resizable window feature for now 0.75 (which will work only as "experimental" feature with `output=opengl` and `glshader=sharp`).

We do want to have it implemented eventually (as it would allow for resizable window with `output=texture` and when using non-opengl texture drivers), but I don't think we'll be able to match scaling quality of a good shader. I do want to replace `sharp` with `pixellate` though (preserving the name, only replacing shader implementation) - it works slightly better (very visible in text modes).